common/tzif.ksy

```diff
@@ -0,0 +1,262 @@
+          for 32-bit time values (used in the version 1 data block). Set to
+          `true` for 64-bit time values (used in the version 2+ data block).
+    seq:
+      - id: transition_times
+        type:
+          switch-on: is_ts64
+          cases:
+            false: s4
+            true: s8
+        repeat: expr
+        repeat-expr: _parent.header.num_transition_times
+        doc: |
+          A series of UNIX leap-time values sorted in strictly ascending order.
+          Each value is used as a transition time at which the rules for
+          computing local time may change. Each time value SHOULD be at least
+          -2**59.
+      - id: transition_types
+        type: u1
+        repeat: expr
+        repeat-expr: _parent.header.num_transition_times
+        doc: |
+          Transition types specifying the type of local time of the
+          corresponding transition time. These values serve as zero-based
+          indices into the array of local time type records. Each type index
+          MUST be in the range [0, `_parent.header.num_local_time_types` - 1].
+      - id: local_time_types
+        type: tz_local_time_type
+        repeat: expr
+        repeat-expr: _parent.header.num_local_time_types
+        doc: Local time type records.
+      - id: tz_designations
+        size: _parent.header.len_tz_designations
+        doc: |
+          Space for a series of NUL-terminated (0x00) time zone designation
+          strings. Two designations MAY overlap if one is a suffix of the other.
+          The character encoding of time zone designation strings is not
+          specified. For interoperbility, time zone designations SHOULD consist
+          of at least three (3) and no more than six (6) ASCII characters from
+          the set of alphanumerics, '-', and '+'. This is for compatibility with
+          POSIX requirements for time zone abbreviations.
+      - id: leap_second_records
+        type: tz_leap_second_record(is_ts64)
+        repeat: expr
+        repeat-expr: _parent.header.num_leap_second_records
+        doc: Leap-second records.
+      - id: is_std_flags
+        # standard/wall indicators
+        type: u1
+        repeat: expr
+        repeat-expr: _parent.header.num_is_std_flags
+        doc: |
+          Values indicating whether the transition times associated with the
+          local time types were specified as standard time or wall-clock time.
+          The value MUST be 0 or 1. A value of one (1) indicates standard time.
+          The value MUST be set to one (1) if the corresponding UT/local
+          indicator is set to one (1). A value of zero (0) indicates wall time.
+          If `_parent.header.num_is_std_flags` is zero (0), all transition times
+          associated with local time types are assumed to be specified as wall
+          time.
+      - id: is_ut_flags
+        # UT/local indicators
+        type: b8
```

Comment:
`b8` is `u1`,  but less efficient.

This probably should be an own type full of bit flags.

Reply:
Actually, that's a bug. It should be `u1` like `is_std_flags`.